src/main/java/net/neoforged/neoforge/fluids/FluidType.java

```diff
@@ -209,6 +217,16 @@ public Rarity getRarity() {
         return this.rarity;
     }
 
+    /**
+     * Returns the Pointed Dripstone drip information of the fluid.
+     *
+     * @return the Pointed Dripstone drip information of the fluid
```

Comment:
Can be simplified using the `{@return}` inline tag.
```suggestion
     * {@return the pointed dripstone drip information of the fluid}
```